pkg/cli/cmd/bicep/publish/publish.go

```diff
@@ -81,6 +81,7 @@ rad bicep publish --file ./redis-test.bicep --target br:ghcr.io/myregistry/redis
 	_ = cmd.MarkFlagRequired("file")
 	cmd.Flags().String("target", "", "remote OCI registry path, in the format 'br:HOST/PATH:TAG'.")
 	_ = cmd.MarkFlagRequired("target")
+	cmd.Flags().Bool("plain-http", false, "allow insecure connections to registry without SSL check")
```

Comment:
Same feedback applies here as I left on the .tsp file. We should be really clear about what this means since it's a security feature. It doesn't need to be simple, it needs to be clear.